website/docs/r/search_service.html.markdown

```diff
@@ -8,79 +8,83 @@ description: |-
 
 # azurerm_search_service
 
-Allows you to manage an Azure Search Service.
+Manages a Search Service.
 
 ## Example Usage
 
 ```hcl
 resource "azurerm_resource_group" "example" {
-  name     = "acceptanceTestResourceGroup1"
-  location = "West US"
+  name     = "example-resources"
+  location = "West Europe"
 }
 
 resource "azurerm_search_service" "example" {
-  name                = "acceptanceTestSearchService1"
+  name                = "example-search-service"
   resource_group_name = azurerm_resource_group.example.name
   location            = azurerm_resource_group.example.location
   sku                 = "standard"
-
-  tags = {
-    environment = "staging"
-    database    = "test"
-  }
 }
 ```
-## Argument Reference
+
+## Arguments Reference
 
 The following arguments are supported:
 
-* `name` - (Required) The name of the Search Service. Changing this forces a new resource to be created.
+* `location` - (Required) The Azure Region where the Search Service should exist. Changing this forces a new Search Service to be created.
+
+* `name` - (Required) The Name which should be used for this Search Service. Changing this forces a new Search Service to be created.
 
-* `resource_group_name` - (Required) The name of the resource group in which to create the Search Service. Changing this forces a new resource to be created.
+* `resource_group_name` - (Required) The name of the Resource Group where the Search Service should exist. Changing this forces a new Search Service to be created.
```

Comment:
Instead of doing alphabetical here should be be consistent with how we order them in the schema/typically right now? name->rg->location ?

Reply:
that's mostly for historical reasons/where there's a larger set of fields - but it's not really giving us any value here, so I think we can ignore that for now